xbmc/guilib/GUIControl.cpp

```diff
@@ -185,7 +186,15 @@ void CGUIControl::DoRender()
       g_graphicsContext.SetStereoFactor(m_stereo);
 
     GUIPROFILER_RENDER_BEGIN(this);
+
+    if (m_hitColor != 0xFFFFFFFF)
```

Comment:
I've made it a CGUIInfoColor because i wanted it to bahave exactly the same as the other colors within skins. This way you can manually specify it in AARRGGBB format or use a pre-defined (info-) color from the skins color theme.